engine/runtime/src/main/java/org/enso/interpreter/caches/Cache.java

```diff
@@ -393,7 +393,11 @@ protected final String computeDigestOfLibrarySources(
    * @return digest used for computing hashes
    */
   protected MessageDigest messageDigest() {
-    return new SHA1.Digest();
+    try {
+      return MessageDigest.getInstance("SHA-1");
```

Comment:
Do we know what implementation of SHA-1 algorithm we get? JDK one or boundycastle one?

Reply:
It's Sun provider, so the JDK one.